docker/README.md

```diff
@@ -0,0 +1,44 @@
+This document is used for generating OWT build and all-in-one OWT running environment image.
+
+## Prequisites
+
+  * Docker is setup in your local environment and related proxy is set so that Docker can access to external network (including Google because OWT will download webrtc related code and patch from Google)
+
+## Build OWT images
+
+1. Run following command to generate Docker image containing OWT source code and building dependencies, and a OWT dist folder will be packed with all OWT running binaries, you can also run OWT service with this image in case you want to modify OWT source code and debug. 
```

Comment:
```suggestion
1. Run following command to generate Docker image containing OWT source code and dependencies. The dist folder will be packed with all OWT running binaries. You can also run OWT service with this image in case you want to modify OWT source code and debug. 
```